package exceptiontopic;

import java.util.Scanner;

public class TestApp1 
{
	public static void main(String[] args) 
	{
		Scanner sc  = new Scanner(System.in);

//		String  name = "royal";
//		System.out.println(name.charAt(10));
//-----------------------------------------------------------------------------		
//		int a[] = new int[5];//a[0],,,a[4]
//		// ArrayIndexOutOfboundException
//		a[5] = 100;
//		System.out.println(a[5]);
//-----------------------------------------------------------------------------		
//		String value1 = "royal";
//		// NumberFormateException
//		int no1 = Integer.parseInt(value1);			
//		System.out.println("No1 : " +  no1);
//-----------------------------------------------------------------------------		
//		String name = null;
		// NullpointerException
//		System.out.println(name.length());
//-----------------------------------------------------------------------------		
//		System.out.println("Enter No1 :");
//		int no1 = sc.nextInt();
//		System.out.println("Enter No2 :");
//		int no2 = sc.nextInt();
		
//		int ans = no1 / no2;//java.lang.ArithmeticException: / by zero
		
//		System.out.println("Addition : " + ans);
//-----------------------------------------------------------------------------		
	}
}
